tar/bsdtar.c

```diff
@@ -1120,6 +1125,7 @@ main(int argc, char **argv)
 	case OPTION_RECOVER_DELETE:
 		missingkey = crypto_keys_missing(CRYPTO_KEYMASK_AUTH_DELETE);
 		break;
+	case OPTION_DUMP_CONFIG:
```

Comment:
Missing `break` here?

Reply:
That's deliberate because I wanted the fallthrough into the `OPTION_PRINT_STATS`, but I should have modified the comment accordingly.

Reply:
Wait, no, you're right that a `break` is better here.